### PR TITLE
Refactor(web-twig): Unify quotes in attributes and print them raw

### PR DESCRIPTION
**Pull request by crishpeen**

### CODE REVIEW

**Comment by literat** [comment]
@crishpeen There are any changed snapshots?

File: packages/web-twig/src/Resources/components/Collapse/Collapse.twig [hunk 1]
```diff
@@ -11,17 +11,17 @@
 {%- set _contentClassName = _spiritClassPrefix ~ 'Collapse__content' -%}
 
 {# Attributes #}
-{%- set _idAttr = _id ? 'id=' ~ _id ~ '' : null -%}
-{%- set _breakpointAttr = _breakpoint ? 'data-breakpoint=' ~ _breakpoint ~ '' : null -%}
-{%- set _parentAttr = _parent ? 'data-parent=' ~ _parent ~ '' : null -%}
+{%- set _idAttr = _id ? 'id="' ~ _id ~ '"' : null -%}
+{%- set _breakpointAttr = _breakpoint ? 'data-breakpoint="' ~ _breakpoint ~ '"' : null -%}
```
Comment (adamkudrna) [inline packages/web-twig/src/Resources/components/Collapse/Collapse.twig:15]: When we are on it, could you please also improve the names of the private `*Attr` variables? I just [described it in another PR](https://github.com/lmc-eu/spirit-design-system/pull/606#discussion_r1064743641).

obrazek

**Comment by crishpeen** [comment]
> @crishpeen There are any changed snapshots?

nope, still the same output, just unified input :)

File: packages/web-twig/src/Resources/components/Collapse/Collapse.twig (hunk 1 above)
Comment (crishpeen) [reply packages/web-twig/src/Resources/components/Collapse/Collapse.twig]: done

**Review by literat — APPROVED** [review]

**Review by tomassychra — APPROVED** [review]

**Review by vmilersky — APPROVED** [review]

**Review by adamkudrna — APPROVED** [review]

**Pull request merged**